### PR TITLE
added validation

### PR DESCRIPTION
**Pull request by YuvHayun**
## Related Issues
fixes: [CIAC-6752](https://jira-hq.paloaltonetworks.local/browse/CIAC-6752)

## Description
Added a check to **is_docker_image_latest_tag** to fail only if docker is at least 3 days old.

### CODE REVIEW

File: demisto_sdk/commands/common/hook_validations/docker.py [hunk 1]
```diff
@@ -112,59 +113,86 @@ def is_docker_image_valid(self):
...
+            self.docker_image_name, self.docker_image_tag
+        )
+        if (
+            not last_updated
+            or three_days_ago
+            and three_days_ago
+            > datetime.strptime(last_updated, "%Y-%m-%dT%H:%M:%S.%fZ")
+        ):
+            return True
+        return False
```
Comment (JudahSchwartz) [inline demisto_sdk/commands/common/hook_validations/docker.py:195]:
```suggestion
        return
            not last_updated
            or three_days_ago
            > datetime.strptime(last_updated, "%Y-%m-%dT%H:%M:%S.%fZ")
```

File: demisto_sdk/commands/common/hook_validations/docker.py [hunk 2]
```diff
@@ -112,59 +113,86 @@ def is_docker_image_valid(self):
...
 
-        return self.is_latest_tag
+    def is_docker_older_than_three_days(self):
+        """
+        Return True if the docker is more than 3 days old.
+
+        Returns:
+            bool: True if the docker is more than 3 days old.
+        """
+        three_days_ago: Optional[datetime] = parse("3 days ago")
```
Comment (JudahSchwartz) [inline demisto_sdk/commands/common/hook_validations/docker.py:184]: why is 3 days ago optional?

File: demisto_sdk/commands/common/hook_validations/docker.py [hunk 3]
```diff
@@ -291,6 +319,37 @@ def find_latest_tag_by_date(tags: list) -> str:
                 latest_tag_name = tag.get("name")
         return latest_tag_name
 
+    @staticmethod
+    @lru_cache(256)
+    def get_docker_image_creation_date(
```
Comment (JudahSchwartz) [inline demisto_sdk/commands/common/hook_validations/docker.py:324]: lets return a date object here

Comment (JudahSchwartz) [inline demisto_sdk/commands/common/hook_validations/docker.py:324]: this fn should be in docker_helper

File: demisto_sdk/commands/common/hook_validations/docker.py [hunk 4]
```diff
@@ -112,59 +113,86 @@ def is_docker_image_valid(self):
...
-                suggested_fix=suggested_fix,
-            ):
-                self.is_latest_tag = False
-
-            else:
-                # if this error is ignored - do print it as a warning
-                self.handle_error(
-                    error_message, error_code, file_path=self.file_path, warning=True
+            self.is_latest_tag = False
+            if self.is_docker_older_than_three_days():
```
Comment (JudahSchwartz) [inline demisto_sdk/commands/common/hook_validations/docker.py:140]: we should log a warning in the else here

File: CHANGELOG.md [hunk 5]
```diff
@@ -15,6 +15,7 @@
 * Fixed an issue in **upload** command where the `--keep-zip` argument did not clean the working directory.
 * Fixed an issue where an extra levelname was added to a logging message.
 * Fixed an issue in **upload** where uploading packs to XSIAM failed due to version mismatch.
+* Added a check to **is_docker_image_latest_tag** to fail the validation on none latest image tag only when current tag is older than 3 days.
```
Comment (JudahSchwartz) [inline CHANGELOG.md:18]:
```suggestion
* Added a check to **is_docker_image_latest_tag** to only fail the validation on non-latest image tag when the current tag is older than 3 days.
```

**Review by JudahSchwartz — APPROVED** [review]
awesome job. This will make life easier

File: demisto_sdk/commands/common/hook_validations/docker.py (hunk 2 above)
Comment (YuvHayun) [reply demisto_sdk/commands/common/hook_validations/docker.py:184]: That's the **parse** return type.

**Comment by JudahSchwartz** [comment]
@YuvHayun looks like you merged CHANGELOG under 15.5 instead of unreleased. Best to fix

**Pull request merged**